packages/design-system/src/components/Alert/Alert.tsx

```diff
@@ -88,7 +88,12 @@ const defaultAnalytics = (heading = '', variation = '') => ({
   },
 });
 
-export class Alert extends React.PureComponent<AlertProps, any> {
+export class Alert extends React.Component<React.HTMLProps<HTMLDivElement> & AlertProps, any> {
```

Comment:
We overlooked omitting the `role` attribute so that we can use role from the component props.
Can replace `export class Alert extends React.Component<React.HTMLProps<HTMLDivElement> & AlertProps, any> {` with:
```
+type OmitProps = 'role';
+export class Alert extends React.Component<
+  Omit<React.ComponentPropsWithRef<'div'>, OmitProps> & AlertProps,
+  any
+> {
```